### ISSUE TITLE
Wake up Issues

### ISSUE THREAD

**Issue by ncrispi** [khronokernel/Opencore-Vanilla-Desktop-Guide#83]
I have the following build:

* MOBO: Z390 Designare 
* Graphics: RX 5700XT
* CPU: Intel i9 9900K
* CPU Fan: CoolMaster M240L
* Wifi/Bluetooth: BCM4360 AC1200
* Case: NZXT H510

I am running OpenCore on 10.15.3 and Everything appears to be working except for constant phantom wakeups (fans turn on, monitor stays off - for about 30 seconds before turning back off) whenever the computer is asleep. 

I have checked for wake reason in terminal with:
log show --style syslog | fgrep "Wake reason"

The errors seem to be the below four from most common:

1) (AppleACPIPlatform) AppleACPIPlatformPower Wake reason: XDCI
2) [powerd:sleepWake] Wake reason: "private” identity: "private”
3) AppleACPIPlatformPower Wake reason: PEG1 PEGP PEG2 PEGP PXSX RP03 PXSX UPSB DSB0 NHI0 DSB1 DSB2 XHC5 DSB4 RP06 PXSX RP07 PXSX RP08 PXSX PXSX RP10 PXSX RP11 PXSX R
4) AppleACPIPlatformPower Wake reason: RTC (Alarm)

WHAT I HAVE FOUND SO FAR:

Above the 'Private' code (2) in the logs I am always getting this right before:

[powerd:sleepWake] AppWoke:com.apple.bluetoothd-blueavengers Reason:BlueAvengers: Toggle beacon state

The XDCI code (1) is actually happening  every time when the computer  shuts off for sleep based off of the time stamps

The other two codes (3,4) I only got once I connected to my iCloud account on the computer.

I have looked at nearly every guide and spent the last 7 days trying to isolate the issue. I have even completely reinstalled the OS and am still getting wakes present in the log.

My USB setup is here:

[https://imgur.com/gallery/5ddljnE](https://imgur.com/gallery/5ddljnE)

Is there anyone that has experienced this that can help, or might know what the issue is?

I’ve done the below per guides:
Disable Hibernation:

sudo pmset -a hibernatemode 0

sudo rm /var/vm/sleepimage

sudo mkdir /var/vm/sleepimage

Disable other hibernation options:

sudo pmset -a standby 0

sudo pmset -a autopoweroff 0

Turned off in settings:

Allow Bluetooth devices to wake up machine

Enable power nap

Allow this devices to wake up for networks

I have also tried recreating EFI folder with bare minimum kexts/ACPI files and only mouse and keyboard USBs active and still getting same wake ups

**Comment by khronokernel** [comment]
The issues tab is not a for troubleshooting setups, it's for reporting issues with the guide itself. If there's an error somewhere in my writing, I'm happy to get an issue opened but please keep system troubleshooting to forums and such

**Comment by ncrispi** [comment]
Hi @khronokernel,

This info would also be very helpful to have in this documentation as well under debugging. There are a few other users on Reddit that I have found with same codes but no one seems to know the root cause / how to resolve. I have gone to various forums and same scenario unfortunately. I figured I would reach out to devs as you all have a way better understanding of Apple kexts/ACPIs than myself and the average user
